tools/dependencies-report/src/test/resources/expectedOutput.txt

```diff
@@ -1,20 +1,20 @@
-dependencyName,dependencyVersion,license
-bundler,1.16.0,MIT
-bundler,1.16.1,MIT
-com.fasterxml.jackson.core:jackson-core,2.7.3,Apache-2.0
-com.fasterxml.jackson.core:jackson-core,2.9.1,Apache-2.0
-com.fasterxml.jackson.core:jackson-core,2.9.4,Apache-2.0
-com.google.errorprone:javac-shaded,9-dev-r4023-3,EPL-1.0
-commons-io:commons-io,2.5,Apache-2.0
-control.js,,MIT
-filesize,0.0.4,MIT
-gradle.plugin.com.github.jk1:gradle-license-report,0.7.1,Apache-2.0
-jar-dependencies,0.3.10,Apache-2.0
-jar-dependencies,0.3.11,MIT
-jar-dependencies,0.3.12,MIT
-jruby-openssl,0.9.20,Apache-2.0
-jruby-openssl,0.9.21,Apache-2.0
-jruby-readline,1.1.1,Apache-2.0
-json-parser,,Apache-2.0
-junit:junit,4.12,Apache-2.0
-org.codehaus.janino:commons-compiler,3.0.8,BSD-3-Clause-Attribution
+dependencyName,dependencyVersion,url,license
```

Comment:
I'm realizing, while we're doing this, we should follow https://github.com/elastic/logstash/issues/8725 exactly and include the revision column, even though we'll never write to it. We should also use the correct column headers at the top there.

If you think that belongs in a separate PR, I'm fine with that as well.

Reply:
I do think a separate PR would be better because that will involve lots of change to the unit tests.